src/WinRT.Runtime/Context.cs

```diff
@@ -7,11 +7,8 @@
 
 namespace WinRT
 {
-    static class Context
+    static partial class Context
     {
-        [DllImport("api-ms-win-core-com-l1-1-0.dll")]
-        private static extern unsafe int CoGetContextToken(IntPtr* contextToken);
-
         [DllImport("api-ms-win-core-com-l1-1-0.dll")]
         private static extern int CoGetObjectContext(ref Guid riid, out IntPtr ppv);

```

Comment:
Can we also fix this signature and drop the runtime marshalling while we're at it? 🙂